pkg/proxy/healthcheck/healthcheck_test.go

```diff
@@ -434,12 +467,14 @@ func testHealthzHandler(server httpServer, status int, t *testing.T) {
 func TestServerWithSelectiveListeningAddress(t *testing.T) {
 	listener := newFakeListener()
 	httpFactory := newFakeHTTPServerFactory()
+	fakeClock := testingclock.NewFakeClock(time.Now())
```

Comment:
Any reason not to use the simpler fake here?

Reply:
No other reason other than: I overlooked it